pkgs/development/python-modules/sphinxemoji/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchFromGitHub, sphinx }:
+
+buildPythonPackage rec {
+  pname = "sphinxemoji";
+  version = "0.2.0";
+  outputs = [ "out" "doc" ];
+
+  src = fetchFromGitHub {
+    owner = "sphinx-contrib";
+    repo = "emojicodes"; # does not match pypi name
+    rev = "v${version}";
+    sha256 = "sha256-TLhjpJpUIoDAe3RZ/7sjTgdW+5s7OpMEd1/w0NyCQ3A=";
+  };
+  pythonImportsCheck = [ "sphinxemoji" ];
```

```suggestion
  pythonImportsCheck = [ "sphinxemoji" ];
```
I'm more familiar with putting this line after `postInstall`